### PR TITLE
feat : added login and sign up page 

### PR DESCRIPTION
**Pull request by Tanjum786**
Created 

- login
- sign up
- forgot password 
pages to e-commerce website
Screenshot from 2022-02-15 23-44-42
Screenshot from 2022-02-15 23-45-10
Screenshot from 2022-02-15 23-46-59

### CODE REVIEW

File: pages/login.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        <div class="login_container">
+            div.
+        </div>
+        <footer>
+            <footer class="footer">
+                <div class="footer-header">
+                    <p><a href="/">Terms & Conditions</a></p>
+                    <p><a href="/">Contact Us</a></p>
+                </div>
+                <ul class="social_media_links">
```
Comment (shubhamkumar648) [inline pages/login.html]: you can add target attributes in all social media  anchor tag , so that it will open in new tab

File: pages/login.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                                class="e_icons fab fa-linkedin"></i></a>
+                    </li>
+                    <li class="list-item-inline">
+                        <a href="https://twitter.com/KadakolTanjum"><i
+                                class="e_icons fa-brands fa-twitter-square fotter_icon"></i></a>
+                    </li>
+                </ul>
+            </footer>
+
+
```
Comment (shubhamkumar648) [inline pages/login.html]: avoid space

File: pages/forgot_pass.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        </nav>
+        <section class="e_login_signup_container">
+            <form>
+            <div class="login_signup_container">
+                <h1 class="login_signup_name">Creat New Password</h1>
+                <label for="userPassword">New Password</label>
+                <input class="input_filed" type="text" name="text" placeholder="Enter Your new password" required>
+                <label for="userPassword">Confirm Password</label>
+                <input class="input_filed" name="password" type="password" placeholder="Enter Your again password" required>
+                <button type="submit" class="login_btn  btn_style" ><a href="/pages/login.html">Reset Password</a></button>            </div>
```
Comment (shubhamkumar648) [inline pages/forgot_pass.html]: use prettier for better indention

**Review by shubhamkumar648 — APPROVED** [review]
consider my comment as a suggestion  rather than code looks good

File: pages/login.html (hunk 1 above)
Comment (Tanjum786) [reply pages/login.html]: okay i will do that thank you

File: CSS/login.css [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    flex-wrap: wrap;
+}
+
+.e_login_signup_container {
+    margin-top: 5rem;
+    justify-content: center;
+}
+
+.login_btn {
+    text-align: center;
```
Comment (anamkaa) [inline CSS/login.css:17]: You can also make use of utility classes for text align center.

**Pull request merged**